manager/manager.py

```diff
@@ -415,11 +415,18 @@ def insert_job(self, job):
         self.job_db_handler.insert_item(job_data)
 
     def query_jobs(self, data=None):
-        items = self.job_db_handler.query_items(data)
+        items, last_key = self.job_db_handler.query_items(data)
         jobs = []
         if items and len(items):
             for item in items:
                 jobs.append(TxJob(item))
+
+        while last_key != None:
```

Comment:
@richmahn - I should probably drop this PR, since I need the fixes in the other PR also which complicates things.